plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/util/InvalidUrlException.java

```diff
@@ -0,0 +1,11 @@
+package io.jenkins.tools.pluginmanager.util;
+
+public class InvalidUrlException extends RuntimeException {
```

Comment:
`IllegalArgumentException` looks like it could be used instead of creating a new exception?